src/app/feedback-form/feedback-form.component.html

```diff
@@ -1,5 +1,5 @@
 <section class="contact-form">
-  <form action="" class="contact-form__container">
+  <form [formGroup]="form" (ngSubmit)="onSubmit()" action="" class="contact-form__container">
```

Comment:
ngSubmit is used when you're using template driven forms. You should use a click on the button itself